osbuild/util/chroot.py

```diff
@@ -33,6 +34,13 @@ def __enter__(self):
                         "sysfs", f"{self.root}/sys"],
                        check=True)
 
+        for d in self._bind_mounts:
+            target_path = os.path.join(self.root, d.lstrip("/"))
+            if not os.path.exists(target_path):
+                    print(f"Making missing chroot directory: {d}")
+                    os.makedirs(target_path)
```

Comment:
I was playing with this over the past few days and  once I tried to bind mount in `/run/osbuild` and it failed.. can you try a test with `/run/osbuild` bind mounted in and see if you see the same failure?

bind mounting in `/run` worked fine, but that may be just because the empty `/run` directory already existed in the target chroot.

Reply:
Same error, and yes. If you see the ` /run `is not created because it already exits,  it fails in the creation of  `/run/osbuild`  
```
/dev/vdb1 on /run/osbuild/inputs/tree type xfs (ro,nosuid,nodev,relatime,seclabel,attr2,inode64,logbufs=8,logbsize=32k,noquota)
```
We would need to have it as rw.
Since `/run`works I don't think it is really an issue.

Reply:
ok - sounds good